.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
     'env': {
         'browser': true,
-        'es2021': true
+        'es2020': true
```

Comment:
Why going down to es2020 ?

Reply:
I think that we do not use any feature of es2021. This way we don't need to enforce a requirement of a most newer browser version, when it really is not needed. We are always in time to raise it when needed, of course.

By the way... precisely i think that some day the minimun browser version declaration must be revised as i think that we are using some newer JS features that these versions do not support.

Reply:
For firefox you can specify on the store wich versions you support; for chrome i think i necer seen any of this feature. Maybe is it automatic ?

Reply:
Maybe seems to be this:

https://developer.chrome.com/docs/extensions/mv3/manifest/minimum_chrome_version/

Anyway... anyhow this is made, sometime we have to choose a suitable minimum browsers version for OGI for the JS features it uses. As if it runs in a browser with no support to these features, funny things may happen... XD